react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+export default class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
```

Comment:
Since you are using a `url` property in `state` it should have a starting value here in your initial state.
```JSX
this.state = {
  url: null
};
```

OR

```JSX
this.state = {
  url: props.urls[0]
};
```